winui/BusyIndicator/Content.md

```diff
@@ -1,12 +1,80 @@
 ---
 layout: post
-title: Content with WinUI BusyIndicator control | Syncfusion
-description: Learn here all about showing the loading progress of the content with Syncfusion WinUI BusyIndicator Control.
+title: Content in WinUI BusyIndicator control | Syncfusion
+description: Learn here all about the loading progress content and its customization available in the Syncfusion WinUI BusyIndicator control.
 platform: WinUI
 control: SfBusyIndicator
 documentation: ug
 ---
 
 # Content in WinUI BusyIndicator
 
-This section explains the content features available in the WinUI BusyIndicator control.
+The BusyIndicator control provides option to set the custom message that indicates the busy status of the control to the users by using the `BusyContent` property.
```

Comment:
The BusyIndicator control provides an option to set the custom message that indicates the busy status of the control to the users by using the `BusyContent` property.

Reply:
Changed the content as mentioned above.